source/org/zfin/marker/agr/BasicDiseaseAnnotationLinkML.java

```diff
@@ -7,7 +7,7 @@
 import lombok.Getter;
 import lombok.Setter;
 import org.alliancegenome.curation_api.model.entities.AffectedGenomicModel;
-import org.alliancegenome.curation_api.model.entities.ontology.EcoTerm;
```

Comment:
Seems like we could just remove this import (along with a bunch of others).  I created PR to this branch if you want to remove those:

https://github.com/ZFIN/zfin/pull/401